Base/res/html/misc/new-tab.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>New Tab</title>
+    <style>
+        #container {
+            display: flex;
+            align-items: center;
+            justify-content: center;
+        }
+
+        #main {
+            text-align: center;
+            display: block;
+            width: 100%;
+            max-width: 400px;
+        }
+
+        #image {
+            image-rendering: pixelated;
+        }
+
+        #input {
+            width: 100%;
+            padding: 5px;
+        }
+
+        #search-buttons {
+            display: flex;
+            justify-content: space-between;
+        }
+
+        button:hover {
+            cursor: pointer;
+        }
+    </style>
+</head>
+<body>
+    <div id="container">
+        <div id="main">
+            <br>
+            <img src="/res/icons/32x32/app-browser.png" width="64" height="64" id="image"><br><br>
+            <form>
+                <input type="search" name="q" id="input"><br><br>
+                <div id="search-buttons">
+                    <button type="button" onclick="search('bing')">Bing</button>
+                    <button type="button" onclick="search('duckduckgo')">DuckDuckGo</button>
+                    <button type="button" onclick="search('frogfind')">FrogFind</button>
+                    <button type="button" onclick="search('github')">GitHub</button>
+                    <button type="button" onclick="search('google')">Google</button>
+                    <button type="button" onclick="search('yandex')">Yandex</button>
+                </div>
+            </form>
+            <br><br>
+            <p>Your user agent is: <b><span id="ua"></span></b></p>
+            <p>This page loaded in <b><span id="loadtime"></span></b> ms</p>
+        </div>
+    </div>
+    <script>
+        document.addEventListener("DOMContentLoaded", function () {
+            document.getElementById("ua").innerHTML = navigator.userAgent;
+            queueMicrotask(function() {
```

Comment:
Why do we need a microtask for this? When `DOMContentLoaded` is fired, the page did "load".